server/modules/MainServer.js

```diff
@@ -21,6 +22,19 @@ let apiV1 = require('api/v1');
 
 const APP_VERSION = require('config').get('APP_VERSION');
 
+function requestFilter(req, propName) {
+  // Avoid writing the authorization token to the log.
+  if (propName === 'headers') {
+    return fp.omit(['authorization'])(req[propName]);
```

Comment:
I'd rather use just normal Lodash for simplicity where it's simpler, but no biggie